keyboards/cannonkeys/ortho48/rules.mk

```diff
@@ -1,6 +1,8 @@
 # project specific files
+VPATH += keyboards/cannonkeys/bluepill
 SRC =	led.c \
-		  ws2812.c
+		  ws2812.c \
```

Comment:
If you're including this, you may want to make sure that you have `RGBLIGHT_CUSTOM_DRIVER = yes` here too.

Reply:
I actually want to rely on the default `rgblight_set` behavior, so I shouldn't define it.

The ws2812.c file acts as if its the driver for ws2812.